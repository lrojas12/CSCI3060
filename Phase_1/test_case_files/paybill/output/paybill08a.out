Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information
Mode in which you wish to login as:
You are currently logged in as an administrator
Enter account holder’sname:
Enter origin account number:
Enter company:
ERROR: The company name is invalid
Must be one of the following: EC, CQ or TV
You have successfully logged out of administrator account

