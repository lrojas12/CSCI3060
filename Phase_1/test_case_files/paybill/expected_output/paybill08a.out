Welcome to Watermelon Banking System
Please login to begin or type in “help” for more information.
Mode in which you wish to login as:
You are currently logged in as an administrator.
Enter account holder’s name:
Enter the origin account number:
Enter company:
Enter amount to pay:
ERROR:The amount entered is not valid.
You have successfully logged out of the administrator account.